Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Script_file_added_to_startup-related_Registry_keys.yml

```diff
@@ -0,0 +1,2980 @@
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: f6a7b9d9-7506-4220-9586-f642211bf61c
+    type: title
+    task:
+      id: f6a7b9d9-7506-4220-9586-f642211bf61c
+      version: -1
+      name: Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "29"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 443,
+          "y": 3728
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 7b81ecc4-b16b-4dd9-9d45-7bda32a67cc0
+    type: condition
+    task:
+      id: 7b81ecc4-b16b-4dd9-9d45-7bda32a67cc0
+      version: -1
+      name: Found any alerts indicating a malicious process execution?
+      description: Determines whether the related alerts indicate malicious activity.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "50"
+      "yes":
+      - "3"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: foundIncidents.sourceBrand
+            iscontext: true
+          right:
+            value:
+              simple: TRAPS
+          ignorecase: true
+        - operator: isEqualString
+          left:
+            value:
+              simple: foundIncidents.name
+            iscontext: true
+          right:
+            value:
+              simple: WildFire Malware
+          ignorecase: true
+        - operator: isEqualString
+          left:
+            value:
+              simple: foundIncidents.name
+            iscontext: true
+          right:
+            value:
+              simple: Local Analysis Malware
+          ignorecase: true
+        - operator: isEqualString
+          left:
+            value:
+              simple: foundIncidents.name
+            iscontext: true
+          right:
+            value:
+              simple: Behavioral Threat
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 838,
+          "y": 890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: 4aec3115-806f-4b66-873a-ab60d142cfc9
+    type: title
+    task:
+      id: 4aec3115-806f-4b66-873a-ab60d142cfc9
+      version: -1
+      name: Block Script File
+      description: Groups all tasks for a specific incident according to the task headers (titles).
```

Comment:
i don't know if this playbook is only for XSIAM, but if it's going to be used in XSIAM 2.x and 3.x, 'incident' might be a problem. XSIAM 2.x has alerts and incidents (and alerts are the rough equivalent of XSOAR incidents). XSIAM 3.x has issues and cases (issues are the equivalent of XSOAR 2.x alerts). I'm not sure what we're doing about this in the playbooks.